src/DependencyInjection/Configuration.php

```diff
@@ -95,13 +95,12 @@ private function addPopulatorSection(ArrayNodeDefinition $rootNode): void
                                 ->normalizeKeys(false)
                                 ->useAttributeAsKey('target')
                                 ->prototype('scalar')
-                                ->isRequired()
                             ->end()
                         ->end()
                     ->end()
                     ->validate()
-                        ->ifTrue(fn (array $c) => empty($c['converter']) && empty($c['property']))
-                        ->thenInvalid('A "converter" and "property" must be defined.')
+                        ->ifTrue(fn (array $c) => empty($c['property']))
```

Comment:
```suggestion
                        ->ifTrue(fn (array $c) => '' !== $c['property'])
```
Shouldn't we avoid empty function everywhere?

Reply:
Depends on the context a bit. We expect an array here, which is not as problematic as e.g. checking strings.

Your suggestion is wrong, by the way:
First, the equivalent for `empty` string would be `'' ===`, not `'' !==`. 
Second, we expect it to be an array, so we should check like: `[] ===`. 
But it also may not be set, so we also need to check with `isset` (which `empty` includes), so the proper replacement would be `!isset($c['property']) || [] === $c['property']` or `[] === ($c['property'] ?? [])`.

So I think in this case `empty` is fine.